src/python/foglamp/storage/tables.py

```diff
@@ -0,0 +1,289 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+"""FogLamp Database Table Definitions
+"""
+
+from sqlalchemy import BigInteger, Boolean, Column, DateTime, ForeignKey, Index, Integer, JSON, MetaData, SmallInteger, String, Table, Time, text
+from sqlalchemy.dialects.postgresql.base import INET, UUID
+from sqlalchemy.dialects.postgresql import JSONB
+
+
+__author__ = "Amarendra Kumar Sinha"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
```

Comment:
I can understand the usefulness of this file for the storage engine which accesses all tables. If this was used by anything else, I would not wish to take the module import overhead of initializing all the variables when my code only uses one of them. Complex module initialization is problematic in general. For example, the 'metadata' variable below is really a temporary but it becomes a global because module initialization can only use globals. It's a minor thing in this particular case but in general becomes a snowball.